projects/core/components/dropdown-box/dropdown-box.component.ts

```diff
@@ -380,10 +380,14 @@ export class TuiDropdownBoxComponent implements AfterViewChecked {
 
         tuiAssertIsElement(host);
 
-        let focusable = tuiGetClosestFocusable(host, previous, root);
+        let focusable = tuiGetClosestFocusable({initial: host, prev: previous, root});
 
         while (focusable !== null && host.contains(focusable)) {
-            focusable = tuiGetClosestFocusable(focusable, previous, root);
+            focusable = tuiGetClosestFocusable({
+                initial: focusable,
+                prev: previous,
```

Comment:
Let's rename `prev` to `previous`. We generally avoid shorting names like that.

Reply:
fixed